src/sweet-loader.js

```diff
@@ -164,7 +168,11 @@ export default class SweetLoader {
     return new Store({});
   }
 
-  compileSource(source: string) {
+  compileSource(source: string, metadata: {}) {
+    let directive = getLangDirective(source);
+    if (directive == null && metadata.enforceLangPragma) {
+      return new SweetModule(List.of());
+    }
```

Comment:
It should definitely not throw.

The idea is to avoid unnecessary expansions of runtime imports since in the vast majority of cases your imports are to non-sweet code.

Logging is reasonable though, I'll add that.